modules/apps/collaboration/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss

```diff
@@ -90,6 +90,12 @@
 			padding-left: 0;
 		}
 
+		.message-content {
+			img {
+				max-width: 100%;
```

Comment:
May not work in Internet Explorer 11 if Flexbox is being used. If display : block is being used IE might pickup on max-width,